orchestra/migrations/0078_todoqa.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
you could recreate 0078 and 0079 so there's only one :)

Reply:
done.